### ISSUE TITLE
image text vibe mismatch

### ISSUE THREAD

**Issue by Parth** [lockbook/lockbook#3654]
The syntax for a link `[text](location)` is very similar to image `![alt text](location)`, however the intent is different. Most places won't render the alt text as it's an accessibility feature (or backup content if the image fails to load for whatever reason). We currently force some default text and render it on anyone who pastes an image. Ideally the alt text would be empty by default. I'm open for us to co-opt this field of data as captions, but ultimately as it won't render that way in most places I'm not sure it's in our user's best interest.
